boards/zub1cg_sbc/dualcam/zub1cg_sbc_dualcam.tcl

```diff
@@ -623,7 +623,7 @@ proc create_hier_cell_CAPTURE_PIPELINE { parentCell nameHier } {
       CONFIG.TUSER_REMAP {tuser[0:0]}] [get_bd_cells axis_subset_converter_1]
 
    # Create instance: mipi_csi2_rx_subsyst_0, and set properties
-   create_bd_cell -type ip -vlnv xilinx.com:ip:mipi_csi2_rx_subsystem:5.4 mipi_csi2_rx_subsyst_0
+   create_bd_cell -type ip -vlnv xilinx.com:ip:mipi_csi2_rx_subsystem:6.0 mipi_csi2_rx_subsyst_0
```

Comment:
I assume these numbers are kernel versions. Would there be any issues if the kernel version is 6.6 and this is set to 6.0 ?

Reply:
This versioning is actually the FPGA IP versioning: https://docs.amd.com/r/en-US/pg232-mipi-csi2-rx/Introduction
Over the years, Xilinx upgrades their IPs, and changed the version number.